src/Compilers/VisualBasic/Portable/Symbols/MissingMetadataTypeSymbol.vb

```diff
@@ -50,7 +50,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Dim containingAssembly As AssemblySymbol = Me.ContainingAssembly
 
                 If containingAssembly.IsMissing Then
-                    Return ErrorFactory.ErrorInfo(ERRID.ERR_UnreferencedAssembly3, containingAssembly.Identity, Me)
+                    Return ErrorFactory.ErrorInfo(ERRID.ERR_UnreferencedAssembly3,
+                                                  CustomSymbolDisplayFormatter.DefaultErrorFormatIfErrorSymbol(containingAssembly.Identity),
```

Comment:
Identity is not a symbol, there is no good reason to call DefaultErrorFormatIfErrorSymbol here.